codalab/client/json_api_client.py

```diff
@@ -7,6 +7,7 @@
 from codalab.common import http_error_to_exception, precondition, UsageError
 from codalab.worker.rest_client import RestClient, RestClientException
 from codalab.worker.download_util import BundleTarget
+from codalab.worker.bundle_service_client import ensure_str
```

Comment:
`ensure_str` should be a general utility function that should be moved to something like `codalab.common`